utils/wrapper.py

```diff
@@ -0,0 +1,37 @@
+import contextlib
+
+import torch
+
+with contextlib.suppress(ModuleNotFoundError):
+    from deepsparse import compile_model
+    from deepsparse.utils import generate_random_inputs
+
+from layers import Detect
+from yolact import FastMaskIoUNet
+
+
+class DeepsparseWrapper:
+    def __init__(self, filepath, cfg, num_cores=None, warm_up_iterations=0,
+                 batch_size=1):
+        self.engine = compile_model(filepath, batch_size=batch_size,
+                                    num_cores=num_cores)
+        self.detect = Detect(cfg.num_classes, bkg_label=0, top_k=cfg.nms_top_k,
+                             conf_thresh=cfg.nms_conf_thresh,
+                             nms_thresh=cfg.nms_thresh)
+        self.maskiou_net = FastMaskIoUNet()
+        if warm_up_iterations:
```

Comment:
let's move warm-up iterations into the core benchmarking logic like we do for the other benchmarking scripts so we can also include post processing

Reply:
Will submit a new pull request with benchmarking moved to a separate script

Reply:
Added